coreos/provision-coreos.sh

```diff
@@ -45,9 +45,13 @@ chmod +x docker-compose
 mkdir -p postgres
 ./docker-compose build
 
-sudo cp nanocloud.architecture.service /etc/systemd/system/nanocloud.architecture.service
+docker pull glyptodon/guacd
```

Comment:
I added in one more commit version number for container I found the information.

guacd version is missing
docker-grand-ambassarod is missing